src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,34 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
+
+void
+ngx_http_lua_ffi_shdict_capacity(ngx_shm_zone_t *zone,
+    size_t *capacity_bytes)
```

Comment:
Better return the `size_t` value directly without using an extra output argument through pointers.

Reply:
done